modules/videoio/test/test_ffmpeg.cpp

```diff
@@ -413,6 +413,9 @@ TEST(videoio, mp4_orientation_meta_auto)
     EXPECT_NO_THROW(cap.open(video_file, CAP_FFMPEG));
     ASSERT_TRUE(cap.isOpened()) << "Can't open the video: " << video_file << " with backend " << CAP_FFMPEG << std::endl;
 
+    // related issue: https://github.com/opencv/opencv/issues/22088
+    EXPECT_EQ(90, cap.get(CAP_PROP_ORIENTATION_META));
```

Comment:
FFmpeg wrapper for Windows is still old (it is not automatically updated).
Please put this check under temporary guard:
```
#ifdef _WIN32  // TODO: FFmpeg wrapper update
```

Reply:
Done